URITemplateTests/URITemplateCases.swift

```diff
@@ -108,12 +108,12 @@ struct Suite {
   let name:String
   let variables:Dictionary<String, AnyObject>
   let cases:[Case]
-  let level:Int
+  var level:Int
```

Comment:
Seems like Swift 1.2 has been a bit more picky, it gets reassigned in `init` on line 121, but as it’s only happening in the there, I can just change it to an else instead.